vscode/webviews/chat/ChatMessageContent/ChatMessageContent.module.css

```diff
@@ -217,3 +217,140 @@ body[data-vscode-theme-kind='vscode-light'] .content pre > code {
     color: var(--vscode-descriptionForeground);
     margin-left: auto;
 }
+
+/* Think Container Styles */
+.thinkContainer {
+    margin-bottom: 1rem;
+    border: 1px solid;
+    border-color: rgba(75, 85, 99, 0.15);
+    border-radius: 0.5rem;
+    overflow: hidden;
+    background-color: rgba(249, 250, 251, 0.5);
+    backdrop-filter: blur(4px);
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+}
+
+:global([data-vscode-theme-kind="vscode-dark"]) .thinkContainer {
+    border-color: rgba(107, 114, 128, 0.2);
+    background-color: rgba(17, 24, 39, 0.5);
+}
+
+.thinkSummary {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0.75rem 1rem;
```

Comment:
I found the Thought Process container is disproportionately large:

![image](https://github.com/user-attachments/assets/9509aeff-0a84-4448-a85b-643f6fcb23ee)

After:

![image](https://github.com/user-attachments/assets/4d1ff8fc-a0ca-4900-b6e2-e03b598c772b)

```suggestion
    padding: 0.15rem 0.5rem;
```